ai-ml/trainium-inferentia/addons.tf

```diff
@@ -80,6 +80,10 @@ module "eks_blueprints_addons" {
     eks-pod-identity-agent = {}
     kube-proxy             = {}
     vpc-cni                = {}
+    amazon-cloudwatch-observability = {
```

Comment:
Does this config provides metrics for Neuron and GPUs as well? Please add some info in the docs on what Neuron metrics are available through CloudWatch agent

Reply:
The add-on deploys DCGM exporter as a daemonset along with the neuron-monitor daemonset. So should work with GPU as well.
However, neuron-monitor ds specifically applies to Neuron instances and collects neuroncore counters and system metrics. Will add a section with details in the vLLM inf2 documentation as a separate PR.